tests/test_params.py

```diff
@@ -173,5 +174,99 @@ def test_bool(self):
         assert opt['bar'] is False
 
 
+class TestAddCmdlineArgs(unittest.TestCase):
+    def test_bad_agent(self):
+        from parlai.core.agents import register_agent
+
+        @register_agent("bad_addcmdlineargs_agent")
```

Comment:
Would it be worth adding a comment here explaining why this is a bad agent and why the later one is a good agent?